src/app/file-explorer.js

```diff
@@ -253,8 +254,6 @@ function fileExplorer (appAPI, files) {
     el.className = css.fileexplorer
     element.parentElement.replaceChild(el, element)
     element = el
-    fileFocus(filepath)
-    appAPI.switchToFile(filepath)
```

Comment:
I think the old behaviour was wrong. Why should I switch to a file just because it was added? The example use-case is that we import a file from github - we should not automatically switch to it. We should only switch when we use the add-button to create a new file, but that should be hanlded by the add button, not by the tree view.